recipe/meta.yaml

```diff
@@ -10,9 +10,8 @@ source:
   sha256: f72f148f54442c6b056bf931dbc34f986fd0c3b0b6b5a58d013c9aef274d0c88
 
 build:
-  number: 0
-  noarch: python
```

Comment:
but didn't you tell me yesterday that this may cause architecture incompatibilities and that generally noarchs are unfavored? Perhaps I misunderstood.

Reply:
When we _rebuild_ a package then we do not remove noarch. But if you have a new package version then, yes, we remove noarch python